.github/workflows/release.yml

```diff
@@ -99,9 +99,15 @@ jobs:
       - name: Create release
         run: |
           gh release create ${{ github.ref }} --generate-notes --prerelease
-          for i in release/*/*; do
-            gh release upload ${RELEASE_NAME} $i
-          done
+          # skip upload if there are no files
+          if stat release/*/* >/dev/null 2>&1; then
+            for i in release/*/*; do
+              gh release upload ${RELEASE_NAME} $i
+            done
+          else
+            echo "No files to upload"
+            exit 0
```

Comment:
Shouldn't this be `exit 1`?

Reply:
No we have crates that don't have binaries.
Each tag and release is per crate.